### DISCUSSION TITLE
Turbo mode fails

### DISCUSSION THREAD

**Discussion by Aoero666** [seerge/g-helper#2396] [Q&A]
If the computer remains in turbo mode before shutting down, it will still be in turbo mode after restarting but its custom CPU fan curve will not take effect. I have to switch it once before it will take effect. This is inconvenient for me. Can you tell me what to do?

**Comment by seerge** [comment]
@Aoero666 what do you understand under "fails" ? How do you verify that?

Do you have AC also running or installed ?

If you think something is not working as it should you need to create a [separate issue, and submit all requested data](https://github.com/seerge/g-helper/issues/new/choose) as without that it's not feasible to assist you further.

Thanks

**Reply by Aoero666** [reply]
When I play games, the fan is at a very low speed, and the CPU fan curve in turbo mode does not follow the settings I set. I had to switch to turbo mode again for the fan speed to go up.
